react/components/Icon.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { string } from 'prop-types'
+import PropTypes from 'prop-types'
```

Comment:
```suggestion
import PropTypes from 'prop-types'

```

Reply:
What's the problem here? 🤔

Reply:
Static import should be separated from relative imports like:

```suggestion
import React from 'react'
import { string } from 'prop-types'

import Use from './Use'
import Svg from './Svg'
import { proptypes } from './propTypes'	import { proptypes } from './propTypes'
```

Reply:
oh! thanks